internal/verifier/verifier_test.go

```diff
@@ -184,8 +258,55 @@ func TestVerifyBuilderImageDigest_DigestNotFound(t *testing.T) {
 	}
 }
 
-func TestVerifyRepoURI_FoundURI(t *testing.T) {
-	got := common.NewProvenanceIR(binarySHA256Digest,
+func TestVerify_NeedsButHasEmptyBuilderImageDigest(t *testing.T) {
+	builderImageSHA256Digest := ""
```

Comment:
Is there a test where builderImage is not set (is nil), but a value is needed for it?
I expect the verification to pass in that case.

Reply:
This case is covered by `TestVerifyHasNoValues`.